canonical-web-and-design/maas-ui
# refactor: update links to the react subnets pages

## Done

- Update the links to subnets and subpages to use the React versions.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Load the UI and click on "Subnets" in the header. It should take you to the React networks page.
- Click on a subnet, vlan, space and fabric and you should get taken to the React pages.
- Click on a VLAN that has a rack controller.
- Click on the controller and you should get taken to the legacy controller page.
- Click on the VLANs tab.
- Click on a fabric, vlan and subnet and you should get returned to the React pages.

## Fixes

Fixes: canonical-web-and-design/app-tribe#677.
